### PR TITLE
DOC-6263: Updated EOL schedule for Corda 4.12

### PR DESCRIPTION
**Pull request by AnnaK-R3**

### CODE REVIEW

File: themes/doks/assets/snippets/corda-4/end-of-life-corda4.md [hunk 1]
```diff
@@ -30,4 +30,6 @@ table th:nth-of-type(5) {
 | **Corda Community Edition 4.10**  | 01/2023             | 01/2025                | 01/2026             | 01/2026            |
 | **Corda Enterprise Edition 4.11** | 11/2023             | 11/2025                | 11/2026             | 11/2026            |
 | **Corda Community Edition 4.11**  | 11/2023             | 11/2025                | 11/2026             | 11/2026            |
+| **Corda Enterprise Edition 4.12** | 06/2024             | 06/2025                | 06/2026             | 06/2026            |
```
Comment (kpishchikR3) [inline themes/doks/assets/snippets/corda-4/end-of-life-corda4.md:33]: Change into 
| **Corda Enterprise Edition 4.12** | 06/2024             | 06/2026                | 06/2027             | 06/2027            |

File: themes/doks/assets/snippets/corda-4/end-of-life-corda4.md [hunk 2]
```diff
@@ -30,4 +30,6 @@ table th:nth-of-type(5) {
...
+| **Corda Community Edition 4.12**  | 06/2024             | 06/2025                | 06/2026             | 06/2026            |
```
Comment (kpishchikR3) [inline themes/doks/assets/snippets/corda-4/end-of-life-corda4.md:34]: Remove this line

**Review by kpishchikR3 — COMMENTED** [review]
please refer to line comments

File: themes/doks/assets/snippets/corda-4/end-of-life-corda4.md (hunk 1 above)
Comment (AnnaK-R3) [reply themes/doks/assets/snippets/corda-4/end-of-life-corda4.md:33]: Updated

File: themes/doks/assets/snippets/corda-4/end-of-life-corda4.md (hunk 2 above)
Comment (AnnaK-R3) [reply themes/doks/assets/snippets/corda-4/end-of-life-corda4.md:34]: Removed

**Review by kpishchikR3 — APPROVED** [review]
lgtm

**Pull request merged**